api/src/pcapi/core/educational/repository.py

```diff
@@ -953,6 +953,36 @@ def get_collective_offer_template_by_ids(offer_ids: list[int]) -> list[education
     return query
 
 
+def get_collective_offer_template_for_playlist(institution_id: int) -> list[educational_models.CollectiveOfferTemplate]:
```

Comment:
[détail] Puisqu'on renvoie une liste de templates, mettons au pluriel ?

```suggestion
def get_collective_offer_templates_for_playlist(institution_id: int) -> list[educational_models.CollectiveOfferTemplate]:
```